Model/Cron.php

```diff
@@ -415,8 +415,7 @@ private function shouldSkipProcessingNotification($notification)
                 // OFFER_CLOSED notifications needs to be at least 10 minutes old to be processed
                 $offerClosedMinDate = new \DateTime('-10 minutes');
                 $createdAt = \DateTime::createFromFormat('Y-m-d H:i:s', $notification['created_at']);
-
-                $minutesUntilProcessing = $createdAt->diff($offerClosedMinDate)->i;
+                $minutesUntilProcessing = ($createdAt->format('U') - $offerClosedMinDate->format('U')) / 60;
```

Comment:
what was the issue with the previous?

Reply:
The usage of ->i is not accurate to calculate the minutes difference between the two DateTime objects created and offer closed . The "i" attribute doesn't take into account the hours and the days in DateInterval.
Using $minutesUntilProcessing = ($createdAt->format('U') - $offerClosedMinDate->format('U')) / 60; Is taking the account the days and the hours

Reply:
ah got it, thanks for the explanation